docs/standalone.rst

```diff
@@ -1,7 +1,9 @@
-Django-Helpdesk Standalone Installation
+Standalone Installation
 =======================================
 
-Installation
+You can find standalone docker images at `djangohelpdesk/standalone:latest <https://hub.docker.com/r/djangohelpdesk/standalone/tags>`_.
+
```

Comment:
Maybe add a text to explain how to install with remote images and also what does the `standalone-extras` image bring in addition to the initial image.

Reply:
I think that most people who would use this standalone image would be inserting it into docker-compose or kubernetes yaml manifest. Not really sure what I should put there, maybe a docker-compose example?

Reply:
Alright I see, so yes just an exemple of commands for docker-compose and why would you be using the extra image.